src/applications/static-pages/cta-widget/components/messages/NoMHVAccount.jsx

```diff
@@ -1,14 +1,28 @@
-import React from 'react';
+import React, { useEffect } from 'react';
 import PropTypes from 'prop-types';
+import recordEvent from '~/platform/monitoring/record-event';
 import CallToActionAlert from '../CallToActionAlert';
 
 const NoMHVAccount = ({
   serviceDescription,
```

Comment:
```suggestion
  recordEventFn = recordEvent,
  serviceDescription,
```

would be the way to get a default prop set